### DISCUSSION TITLE
:Select command will create a new csv.txt file everything and close the main csv buffer

### DISCUSSION THREAD

**Discussion by lanbufan** [mechatroner/rainbow_csv#50] [Q&A]
Hi,

Cool plugin. But I am wondering if there is a way to create a CSV when using Select command, and also how to prevent the main buffer to close?

Thanks.

Francois

**Comment by mechatroner** [comment]
Hi, thanks for the feedback!
It shouldn't close the previous buffer - it opens a new one, but the original should also still be open, Have you tried `:b#` command (go to the previous buffer) and other buffer navigations commands https://vi.stackexchange.com/a/14412/21149 ? Also if you run `:ls` (list open buffers) does the list have the original file?
